frontend/src/routes/manage/Video/Details.tsx

```diff
@@ -72,9 +73,63 @@ const Page: React.FC<Props> = ({ event }) => {
     </>;
 };
 
+type TimePickerProps = {
+    setTimestamp: (newTime: string) => void;
+  }
+
+export const TimePicker: React.FC<TimePickerProps> = ({ setTimestamp }) => {
+    const { t } = useTranslation();
+    const [time, setTime] = useState<string>("00:00:00");
+
+    const formatTime = (timeValue: string): string => {
+        const [hours, minutes, seconds] = timeValue.split(":");
+        let formattedTime = "";
+        if (hours && hours !== "00") {
+            formattedTime += hours + "h";
+        }
+        if (minutes && minutes !== "00") {
+            formattedTime += minutes + "m";
+        }
+        if (seconds && seconds !== "00") {
+            formattedTime += seconds + "s";
+        }
+
+        return formattedTime;
+    };
+
+    useEffect(() => {
+        setTimestamp(formatTime(time));
+    }, [time]);
+
+    return <div css={{ paddingLeft: 2 }}>
+        <label css={{ color: COLORS.neutral90, fontSize: 14 }}>
+            {t("manage.my-videos.details.set-time")}
+        </label>
+        <input
+            value={time}
+            type="time"
+            step="1"
```

Comment:
Unfortunately, I think `<input type="time"` is not the right tool. From reading the MDN documentation it really seems like it's designed to pick a "time of day" and not a "duration" or "timestamp". In my Firefox and Chrome, it's shown like this:

![image](https://github.com/elan-ev/tobira/assets/7419664/e2751ac5-bec1-4fcf-9b62-aa23e4677bf3)

MDN also says that it switches to a "12h clock" based on locale. (Not that I configured my OS or browsers to use 12h, I much prefer 24h; but that's a completely separate Linux issue :P). And I haven't found a way to disable this 12h thing.

So yeah, I'm afraid we have to manually build something here. I'm not sure if we should use 3 `<input type="text"` with colons in between or just one `<input` that includes the colons in its value?